core/src/components/checkbox/checkbox.ionic.scss

```diff
@@ -1,4 +1,17 @@
+@import "./checkbox";
 @import "./checkbox.ionic.vars";
 
 // Ionic Checkbox
-// --------------------------------------------------
\ No newline at end of file
+// --------------------------------------------------
+
+:host {
+  // Border
+  --border-radius: #{$checkbox-ionic-border-radius};
```

Comment:
Discussed internally with Liam & Brandy. You will notice the naming convention here is different than iOS/MD. We've decided to remove the `-icon` from the name. It didn't make sense. 

We left iOS/MD alone, this only applies to the ionic theme.